src/serializers/graph_extensions.c

```diff
@@ -82,9 +83,11 @@ void Serializer_Graph_SetNodeLabels
 
 		GxB_Vector_diag(v, m, 0, NULL);
 
-		GxB_Col_subassign(node_labels_m, NULL, NULL, v, GrB_ALL, 0, i, NULL);
+		GxB_Row_subassign(node_labels_m, NULL, NULL, v, i, GrB_ALL, 0, NULL);
 	}
 
+	GrB_transpose(node_labels_m, NULL, NULL, node_labels_m, NULL);
```

Comment:
Perfect, we should have realized `GxB_Col_subassign` is expensive when `node_labels_m` is `CSR`

Reply:
> can we add a benchmark guarding from future possible performance degradation ?

![image](https://user-images.githubusercontent.com/5832149/159428400-64fb7b87-6051-4b0a-a3be-e1f3004c8c30.png)

@swilly22 we keep track of a "generic" rdb load timetaken over time that we can use to create the comparison tables but I believe we need a more advanced way of keeping track of this plus the setups with replication enabled ( that we don't have on performance automation still ). 
I will do the required changes on the performance package parts and then we can be more assertive in making sure te rdb load time and full sync time, etc... are kept at a constant level or improved